src/openvic-simulation/country/Country.hpp

```diff
@@ -63,11 +63,16 @@ namespace OpenVic {
 		const unit_names_map_t PROPERTY(unit_names);
 		const bool PROPERTY_CUSTOM_PREFIX(dynamic_tag, is);
 		const government_colour_map_t PROPERTY(alternative_colours);
+		colour_t PROPERTY(primary_unit_colour);
+		colour_t PROPERTY(secondary_unit_colour);
+		colour_t PROPERTY(tertiary_unit_colour);
```

Comment:
Shouldn't these be const?

Reply:
Cannot be const as it is added outside the constructor